pyoptsparse/pyOpt_variable.py

```diff
@@ -82,12 +82,7 @@ def __str__(self) -> str:
         if self.type == "d":
             res += "	 "
             res += str(self.name).center(15)
-            res += "{:>25}{:20f} {:14.2e} {:12.2e} \n".format(
```

Comment:
why did you change `self.choices[int(self.value)]` into just `self.value`?

Reply:
Good question. It seems like `self.value` is already an integer from above, but at the same time I'm also somewhat certain this stuff is broken since `value` and `self.value` are not the same (in the `__init__`). We do not have any tests for integer DVs I think.

Reply:
Yeah but I think in the first case `self.choices` was a list, and now it's replaced by an integer? Anyway, if we don't have tests and we don't have working examples with this bit of code this is very hard to debug

Reply:
Yeah you know what I'll just put this back so this PR is not making any syntax changes. I know we should have tests for this but it's very low on my priority list.

Reply:
yeah I am on the same page, this is not urgent now